pkg/bundle/internal/target/target.go

```diff
@@ -396,3 +427,18 @@ func (b *Data) Populate(pool *util.CertPool, formats *trustapi.AdditionalFormats
 	}
 	return nil
 }
+
+// Calculate new password hash annotations from the given additional formats
+// keys and passwords
+func calcNewHashAnnotations(bundle *trustapi.AdditionalFormats) map[string]string {
```

Comment:
```suggestion
func truststorePasswordAnnotations(bundle *trustapi.AdditionalFormats) map[string]string {
```

Just a suggestion. To me, the current name doesn't have the correct context. The term "new" is not relevant, and "HashAnnotations" is too generic.

Reply:
I agree. Fixed